iroh/examples/sync.rs

```diff
@@ -349,6 +349,50 @@ async fn handle_command(
         }
         Cmd::Stats => get_stats(),
         Cmd::Fs(cmd) => handle_fs_command(cmd, doc).await?,
+        Cmd::Hammer {
+            prefix,
+            threads,
+            count,
+            size,
+        } => {
+            println!(
+                "> Hammering with prefix {prefix} for {threads} x {count} messages of size {size} bytes",
+                prefix = prefix,
+                threads = threads,
+                count = count,
+                size = size,
+            );
+            let mut bytes = vec![0; size];
+            bytes.fill(97);
```

Comment:
If we also want to test content downloading with this (do we?) maybe let's set their content to an increasing number or add an option for that behavior. Can also come later though.